src/main/java/uk/gov/companieshouse/api/accounts/model/rest/smallfull/notes/offbalancesheetarrangements/OffBalanceSheetArrangements.java

```diff
@@ -0,0 +1,50 @@
+package uk.gov.companieshouse.api.accounts.model.rest.smallfull.notes.offbalancesheetarrangements;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import uk.gov.companieshouse.api.accounts.model.rest.Note;
+import uk.gov.companieshouse.api.accounts.validation.CharSetValid;
+import uk.gov.companieshouse.charset.CharSet;
+
+import javax.validation.constraints.Size;
+import java.util.Objects;
+
+@JsonInclude(JsonInclude.Include.NON_NULL)
+public class OffBalanceSheetArrangements extends Note {
+
+    private static final int MAX_FIELD_LENGTH = 20000;
```

Comment:
Should we move this to the Note.java class so that it can be reused when you refactor the other notes ?

Reply:
Good suggestion. Moved.